### PR TITLE
Replace CommonPlan with inline definitions in SubscriptionOrderPlan

### PR DESCRIPTION
**Pull request by swilczynski**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Storefront/StorefrontPlan.yaml [hunk 1]
```diff
@@ -1,12 +1,107 @@
...
+      Number of times the plan is modified.
+      Compare this value with materialized subscription item revision values.
+  createdTime:
+    $ref: ../CreatedTime.yaml
+  updatedTime:
+    $ref: ../UpdatedTime.yaml
+  isTrialOnly:
+    type: boolean
+    description: Specifies if a plan is a trial that does not have recurring instructions.
+    readOnly: true
```
Comment (ashkarpetin) [inline openapi/components/schemas/Storefront/StorefrontPlan.yaml:85]: I think this is not present in the Storefront plan resource

Comment (ashkarpetin) [reply openapi/components/schemas/Storefront/StorefrontPlan.yaml:85]: `trial` seems missing

File: openapi/components/schemas/Plans/PlanTypes/TrialOnlyPlan.yaml [hunk 2]
```diff
@@ -6,9 +6,81 @@ required:
...
+  description:
+    type: string
+    description: |-
+      Plain-text description of the plan.
+      This field accepts plain-text only.
+  richDescription:
+    type: string
+    description: |-
+      Rich-text description of the plan.
+      This field accepts rich text formatting,
```
Comment (nanobash) [inline openapi/components/schemas/Plans/PlanTypes/TrialOnlyPlan.yaml:30]: I think this should be a one liner:

```suggestion
      This field accepts rich text formatting, such as: bold, underline, italic, and hyperlinks.
```

File: openapi/components/schemas/Plans/PlanTypes/OneTimeSalePlan.yaml [hunk 3]
```diff
@@ -1,4 +1,101 @@
...
+    type: string
+    description: |-
+      Plain-text description of the plan.
+      This field accepts plain-text only.
+  richDescription:
+    type: string
+    description: |-
+      Rich-text description of the plan.
+      This field accepts rich text formatting,
+      such as: bold, underline, italic, and hyperlinks.
```
Comment (nanobash) [inline openapi/components/schemas/Plans/PlanTypes/OneTimeSalePlan.yaml:30]: I think this should be a one liner:

```suggestion
      This field accepts rich text formatting, such as: bold, underline, italic, and hyperlinks.
```

File: openapi/components/schemas/Plans/PlanTypes/SubscriptionOrderPlan.yaml [hunk 4]
```diff
@@ -1,33 +1,117 @@
...
+    type: string
+    description: |-
+      Plain-text description of the plan.
+      This field accepts plain-text only.
+  richDescription:
+    type: string
+    description: |-
+      Rich-text description of the plan.
+      This field accepts rich text formatting,
+      such as: bold, underline, italic, and hyperlinks.
```
Comment (nanobash) [inline openapi/components/schemas/Plans/PlanTypes/SubscriptionOrderPlan.yaml:30]: A nitpick, I think this should be a one liner:

```suggestion
      This field accepts rich text formatting, such as: bold, underline, italic, and hyperlinks.
```

File: openapi/components/schemas/Plans/PlanTypes/TrialOnlyPlan.yaml [hunk 5]
```diff
@@ -6,9 +6,81 @@ required:
...
+      such as: bold, underline, italic, and hyperlinks.
```
Comment (nanobash) [inline openapi/components/schemas/Plans/PlanTypes/TrialOnlyPlan.yaml:31]: A nitpick, I think this should be a one liner:

```suggestion
      This field accepts rich text formatting, such as: bold, underline, italic, and hyperlinks.
```

File: openapi/components/schemas/Storefront/StorefrontPlan.yaml [hunk 6]
```diff
@@ -1,12 +1,107 @@
...
+    type: string
+    description: |-
+      Plain-text description of the plan.
+      This field accepts plain-text only.
+  richDescription:
+    type: string
+    description: |-
+      Rich-text description of the plan.
+      This field accepts rich text formatting,
+      such as: bold, underline, italic, and hyperlinks.
```
Comment (nanobash) [inline openapi/components/schemas/Storefront/StorefrontPlan.yaml:29]: A nitpick, I think this should be a one liner:

```suggestion
      This field accepts rich text formatting, such as: bold, underline, italic, and hyperlinks.
```

File: openapi/components/schemas/Storefront/StorefrontPlan.yaml (hunk 1 above)
Comment (swilczynski) [reply openapi/components/schemas/Storefront/StorefrontPlan.yaml]: Thanks for noticing, I just pushed a fix

**Review by ashkarpetin — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Review by VladimirBerdnik — APPROVED** [review]

**Pull request merged**